CHANGELOG.md

```diff
@@ -16,6 +46,7 @@ This project adheres to the Node [default version scheme](https://docs.npmjs.com
 - Public: Added a troubleshooting section to the documentation with details about solving CSP related issues
 - UI: Added selfie intro screen
 - UI: Option to send cross device secure link using QR code (**Note:** *changes introduced with this UI update include possible breaking changes for integrators with custom translations or copy*)
+- Public: Added a `userAnalyticsEvent` to existing analytics calls for integrators to listen for.
```

Comment:
@kopijunkie @bartzet I can see that this line might have been added under the wrong release header by mistake. I will move it under `5.8.0`